spec/views/uploads/download/index.html.erb_spec.rb

```diff
@@ -1,5 +0,0 @@
-require "rails_helper"
-
-RSpec.describe "download/index.html.erb", type: :view do
```

Comment:
There were a ton of these extraneous test files created when I created controllers the generator. We're never going to use them and they just clog up our rspec.